src/lib.rs

```diff
@@ -189,22 +189,26 @@ impl CertificateResult {
 /// Certificate paths from `SSL_CERT_FILE` and/or `SSL_CERT_DIR`.
 struct CertPaths {
     file: Option<PathBuf>,
-    dir: Option<PathBuf>,
+    dir: Option<Vec<PathBuf>>,
```

Comment:
Sorry, I meant to suggest that this should probably also just be `Vec<PathBuf>`, since the `Option` can be adequately expressed by an empty `Vec`.

Reply:
> The problem with this approach is that let's say SSL_CERT_DIR is an empty string, then you wont be able to
> distinguish it, from non-existing SSL_CERT_DIR env.

Okay, that makes sense.

Reply:
I don't think `SSL_CERT_DIR=` is a useful case. I assume we'd take "", load nothing from there, and end up with no certificates.

FWIW it seems openssl treats unset and empty values differently -- unset ones obviously use the default, while empty values raise an error. But the immediate caller says "We ignore errors, in case the directory doesn't exist".